src/views/HomePage.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <div id="app">
+    <HelloWorld msg="Welcome to Your Vue.js App" />
+    <TopMenu></TopMenu>
+    <WelcomeWrapper></WelcomeWrapper>
+    <WelcomeMovieAbout></WelcomeMovieAbout>
+    <WelcomeMovieWrapper></WelcomeMovieWrapper>
+    <ContactUs></ContactUs>
+    <main>
+      <router-view />
+    </main>
```

Comment:
Does HomePage have any nested routes or do you plan to have any?
If not then we don't need to place the `router-view` here.